faksam/Maintenance-Tracker-App
# [Feature #158202858] change account password

#### What does this PR do?
* Update Tests
* Update route to include /account/password
* Update account controller
* Fix page reloading issue
* Create helper function to validate password input
* Create helper funtion to confirm if current password is validFeature #158202858] change account password

#### Description of Task to be completed?
* N/A
#### How should this be manually tested?
* Visit [M-T-A](https://maintenancetracker.herokuapp.com/) login as Admin or User
* Go to My Account 
* Click Change Password
* Save New Account Detail
OR
* Git Clone [MTA] - `git clone (https://github.com/faksam/Maintenance-Tracker-App.git)`
* Run - `npm install`
* Run - `npm run db:migrate`
* Run - `npm start` or `npm run dev`
* Open postman and navigate to `http://localhost:3456/api/v1/` to access application API
* Login as admin or user at `http://localhost:3456/api/v1/auth/login`
* Make a PUT request to `http://localhost:3456/api/v1/users/account/password`  
* Then the user account detail should be updates

#### What are the relevant pivotal tracker stories?(if applicable)
* Pivotal Tracker Story ID's - [158202858](https://www.pivotaltracker.com/story/show/158202858)
